Servers/UI/OJS.Servers.Ui/ClientApp/src/utils/urls.ts

```diff
@@ -307,4 +313,5 @@ export {
     getSubmissionsUrl,
     encodeUsernameAsUrlParam,
     decodeUsernameFromUrlParam,
+    generateContestLinkUrl,
```

Comment:
This does not follow the naming convention in the file. Also the name `contestLinkUrl` is too generic, better specify what is this link for.